techniques/general/G206.html

```diff
@@ -15,7 +15,7 @@
       </section>
       <section class="example">
          
-            <p>A financial spreadsheet is online. It includes text explaining changes in the housing market in January. Off-screen to the right, there is a column with an explanation of changes to the market in September. The user can horizontally scroll to the September area and read each line of text without any further scrolling when the window size is maximized. </p>
+            <p>A financial spreadsheet is online. It includes text explaining changes in the housing market in January. Off-screen to the right, there is a column with an explanation of changes to the market in September. The user can horizontally scroll to the September area and read each line of text without any further scrolling when the viewport (i.e., window) is maximized. </p>
```

Comment:
Do we need "(i.e., window)"? We haven't used it anywhere else.

Reply:
In this case, since "window" is used in the original, I think cueing the inexperienced user that "viewport" is _basically_ a synonym for _window_ here helps.